nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,237 @@
+            try:
+                for seed in self.seeds:
+                    assert(len(seed) == 3)
+            except Exception as e:
+                if self.verbose > 0:
+                    print('Seeds not valid, error' + str(e))
+                raise ValueError('Seeds must be a list of triplets of '
+                                 'coordinates in native space.')
+            self.seeds_ = self.seeds
+        return self
+
+    def fit_transform(self, imgs, confounds=None):
+        return self.fit().transform(imgs, confounds=confounds)
+
+    def _check_fitted(self):
+        if not hasattr(self, "seeds_"):
+            raise ValueError('It seems that %s has not been fitted. '
+                             'You must call fit() before calling transform().'
+                             % self.__class__.__name__)
+
+    def transform(self, imgs, confounds=None):
+        """Extract signals from images.
+
+        Parameters
+        ==========
+        imgs: Niimg-like object
+            See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
+            Images to process. It must boil down to a 4D image with scans
+            number as last dimension.
+
+        confounds: array-like, optional
+            This parameter is passed to signal.clean. Please see the related
+            documentation for details.
+            shape: (number of scans, number of confounds)
+
+        Returns
+        =======
+        signals: 2D numpy.ndarray
+            Signal for each region.
+            shape: (number of scans, number of regions)
+
+        """
+        self._check_fitted()
+
+        logger.log("loading images: %s" %
+                   _utils._repr_niimgs(imgs)[:200], verbose=self.verbose)
+        imgs = _utils.check_niimgs(imgs)
+
+        if self.smoothing_fwhm is not None:
+            logger.log("smoothing images", verbose=self.verbose)
+            imgs = self._cache(image.smooth_img, func_memory_level=1)(
+                imgs, fwhm=self.smoothing_fwhm)
+
+        logger.log("extracting region signals", verbose=self.verbose)
+        signals = self._cache(
+            _signals_from_seeds, func_memory_level=1)(
+                self.seeds_, imgs, radius=self.radius)
+
+        logger.log("cleaning extracted signals", verbose=self.verbose)
+        signals = self._cache(signal.clean, func_memory_level=1
+                                     )(signals,
+                                       detrend=self.detrend,
+                                       standardize=self.standardize,
+                                       t_r=self.t_r,
+                                       low_pass=self.low_pass,
+                                       high_pass=self.high_pass,
+                                       confounds=confounds)
+        return signals
+
+    def inverse_transform(self, signals):
+        raise ValueError('Inverse transformation has no sense for seeds.')
```

Comment:
I'd rather have the method simply not present. That way it is possible to test if an object implements inverse_transform or not using hasattr.